pages/api/moon.svg/index.ts

```diff
@@ -1,22 +1,25 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
 import type { NextApiRequest, NextApiResponse } from 'next';
 
-import { getMoonPhases } from '../../../server/utils/moon';
-import { createMoon } from '../../../server/utils/svg';
+import type { Theme } from 'server/theme/types';
+
+import createMoonFuncMap from 'server/theme';
+import { isTheme } from 'server/utils/type';
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse,
 ) {
-  const { date, size = '100', round } = req.query;
+  const { date, size = '100', theme } = req.query;
```

Comment:
제가 짠 코드 부분이라 셀프리뷰이긴한데

기존 api와의 호환성을 유지하기 위해서 theme의 default 값이 들어있어야할 것 같아요.
```suggestion
  const { date, size = '100', theme = 'basic' } = req.query;
```